Adjust/adjust/src/main/java/com/adjust/sdk/ActivityHandler.java

```diff
@@ -756,6 +756,14 @@ private void launchDeeplink(ResponseData responseData, Handler handler) {
         }
 
         Uri location = Uri.parse(deeplink);
+
+        if (adjustConfig.onDeeplinkResponseListener != null) {
```

Comment:
This listener is synchronous, so the sdk is waiting until the execution and it is running on the background.
Should it run in the main thread and be async?